Error: CliError { error: Some(failed to get `clap` as a dependency of package `ci v0.0.0 (/srv/t2/src.librsvg.mipsel.250901.202
112.13514/librsvg-2.61.0/ci)`

Caused by:
    0: failed to load source for dependency `clap`
    1: Unable to update registry `crates-io`
    2: failed to update replaced source registry `crates-io`
    3: failed to parse manifest at `/srv/t2/src.librsvg.mipsel.250901.202112.13514/librsvg-2.61.0/vendor/cfg-expr/Cargo.toml`
    4: feature `edition2024` is required

       The package requires the Cargo feature called `edition2024`, but that feature is not stabilized in this version of Cargo
 (1.81.0).
       Consider trying a more recent nightly release.
       See https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#edition-2024 for more information about the status o
f this feature.
       ), exit_code: 101 }

--- librsvg-2.61.0/vendor/cfg-expr/Cargo.toml.vanilla	2025-09-01 20:25:17.126052367 +0200
+++ librsvg-2.61.0/vendor/cfg-expr/Cargo.toml	2025-09-01 20:27:59.706839942 +0200
@@ -10,7 +10,7 @@
 # See Cargo.toml.orig for the original contents.
 
 [package]
-edition = "2024"
+edition = "2021"
 rust-version = "1.85.0"
 name = "cfg-expr"
 version = "0.20.2"
